uk.co.compucorp.civicrm.tasksassignments/js/CRM/Form/Options.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+(function init () {
+  var componentIdRow = CRM.$('select#component_id').parents('tr');
+  var tableBody = componentIdRow.parents('tbody');
+
+  // Move the category field to the top of the form:
+  componentIdRow.detach().prependTo(tableBody);
```

Comment:
I am wondering if `appendTo` would work here https://stackoverflow.com/questions/1279957/how-to-move-an-element-into-another-element?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa could you try please?